src/Microsoft.Diagnostics.ExtensionCommands/Host/RuntimesCommand.cs

```diff
@@ -46,21 +52,36 @@ public override void Invoke()
                 throw new DiagnosticsException("Cannot specify both -netfx and -netcore options");
             }
 
+            bool flush = false;
+            if (UseContractReader.HasValue)
+            {
+                SettingsService.UseContractReader = UseContractReader.Value;
+                flush = true;
+            }
+
+            if (ForceUseContractReader.HasValue)
+            {
+                SettingsService.UseContractReader = ForceUseContractReader.Value;
+                SettingsService.ForceUseContractReader = ForceUseContractReader.Value;
+                flush = true;
+            }
```

Comment:
I don't understand why this is needed. From the command line it makes sense, but after parsing is done, why doesn't this just set `SettingsService.UseContractReader` and use that for everything?

Reply:
Currently UseContractReader enables the CLRMA API usage for watson but not the rest of SOS (uses the regular DAC).  If ForceUseContractReader is true, the CDAC is used everywhere.